package.json

```diff
@@ -38,34 +40,33 @@
     "start:electron": "SILEX_ELECTRON=true electron ."
   },
   "dependencies": {
-    "body-parser": "1.13.3",
+    "bluebird": "*",
```

Comment:
I don't think that's what you want. Even with a `package-lock` this file should be human readable (the lock clearly isn't).

Basically, we talked about express version and it's now very difficult to know which one is used here.

Reply:
i agree but one of the dependencies has a dependency that breaks the production server but works on localhost. I have no idea which and it is really hard to test since I need to deploy every time (and break the server on preprod.silex.me)

Reply:
ok I decided to give it one more chance, I fixed all versions to the latest available and it seems to work on preprod.silex.me

:+1: